src/test/java/persistence/sql/ddl/DataDefinitionLanguageGeneratorTest.java

```diff
@@ -0,0 +1,183 @@
+        assertThat(tableCreator.getTableName().getName()).isEqualTo("ClassWithoutTable");
+    }
+
+    @Test
+    @DisplayName("@Table이 있어도 name이 없다면 TableName 은 클래스 이름이다.")
+    void TestClassNameWithTableButNoName() {
+        TableCreator tableCreator = dataDefinitionLanguageGenerator.generateTableCreatorWithClass(
+            ClassWithTableButEmptyName.class);
+        assertThat(tableCreator.getTableName().getName()).isEqualTo("ClassWithTableButEmptyName");
+    }
+
+    @Test
+    @DisplayName("@Table이 있고 name이 있다면 TableName 은 지정한 이름이다.")
+    void TestClassNameWithTableAndName() {
+        TableCreator tableCreator = dataDefinitionLanguageGenerator.generateTableCreatorWithClass(
+            ClassWithTableAndName.class);
+        assertAll(
+            () -> assertThat(tableCreator.getTableName().getName()).isNotEqualTo("ClassWithTableAndName"),
+            () -> assertThat(tableCreator.getTableName().getName()).isEqualTo("testClass")
+        );
+    }
+
+    @Test
+    @DisplayName("@Column 이 지정되지 않았다면 클래스 필드명이 테이블 필드명이 된다")
+    void testFieldNameWithoutColumn() {
+        TableCreator tableCreator = dataDefinitionLanguageGenerator.generateTableCreatorWithClass(
+            ColumnTestClass.class);
+
+        List<DatabaseField> databaseFields = tableCreator.getFields().getDatabaseFields();
+        assertThat(databaseFields.stream().filter(it -> "noColumn".equals(it.getName())).count()).isEqualTo(1L);
+    }
+
+    @Test
+    @DisplayName("@Column 이 있어도 name이 빈 값이면 클래스 필드명이 테이블 필드명이 된다")
+    void testFieldNameWithColumnButNoName() {
+        TableCreator tableCreator = dataDefinitionLanguageGenerator.generateTableCreatorWithClass(
+            ColumnTestClass.class);
+
+        List<DatabaseField> databaseFields = tableCreator.getFields().getDatabaseFields();
+        assertThat(databaseFields.stream().filter(it -> "columnWithoutName".equals(it.getName())).count()).isEqualTo(
+            1L);
+    }
+
+    @Test
+    @DisplayName("@Column 이 있어도 name이 있다면 테이블 필드명이 된다")
+    void testFieldNameWithColumnAndName() {
+        TableCreator tableCreator = dataDefinitionLanguageGenerator.generateTableCreatorWithClass(
+            ColumnTestClass.class);
+
+        List<DatabaseField> databaseFields = tableCreator.getFields().getDatabaseFields();
+        assertAll(
+            () -> assertThat(databaseFields.stream().filter(it -> "columnWithName".equals(it.getName())).count()).isEqualTo(0L),
+            () -> assertThat(databaseFields.stream().filter(it -> "column".equals(it.getName())).count()).isEqualTo(1L)
+        );
+    }
+
+
+    @Test
+    @DisplayName("nullable 기본값은 true이다")
+    void testDefaultNullable() {
+        TableCreator tableCreator = dataDefinitionLanguageGenerator.generateTableCreatorWithClass(
+            ColumnTestClass.class);
+
+        List<DatabaseField> databaseFields = tableCreator.getFields().getDatabaseFields().stream().filter(it -> "defaultNullableColumn".equals(it.getName())).toList();
```

Comment:
```suggestion
        List<DatabaseField> databaseFields = tableCreator.getFields().getDatabaseFields().stream().filter(it -> "defaultNullableColumn".equals(it.getName())).?;
```
![스크린샷 2023-10-20 오후 8 29 35](https://github.com/next-step/jpa-query-builder/assets/12059592/5d9e191c-a64d-4f52-8fa1-6d7e1e91f163)

`toList();` 는 메소드는 java 16? 부터 지원할꺼에요~
현과정은 메일공지드린것 처럼 java 11버전 기준으로 진행하고 있으니 11버전으로 변경 후 진행부탁드려요~